webui/react/src/e2e/fixtures/global-fixtures.ts

```diff
@@ -155,6 +154,7 @@ export const expect = baseExpect.extend({
     } catch (e: any) {
       matcherResult = e.matcherResult;
       pass = false;
+      const actualTitle = await page.title();
```

Comment:
it's actually better to grab this title as late as possible. It's a mistake I made from the PR that introduced the feature. by the way, the reason we have this `toHaveDeterminedTitle` is to remove the determined logic from basePage, which will eventually become a shared module